From ee52b931335fdcdf32283482558008d865bdd4b5 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Fri, 17 Mar 2017 13:23:33 +0100
Subject: [PATCH] tc: flower: remove references to eth_type in manpage

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1422629
Upstream Status: iproute2.git commit 730381fede498

commit 730381fede4989901911bc981a99079a1d84ca6a
Author: Simon Horman <simon.horman@netronome.com>
Date:   Fri Dec 2 14:59:43 2016 -0800

    tc: flower: remove references to eth_type in manpage

    Remove references to eth_type and ether_type (spelling error) in
    the tc flower manpage.

    Also correct formatting of boldface text with whitespace.

    Cc: Paul Blakey <paulb@mellanox.com>
    Signed-off-by: Simon Horman <simon.horman@netronome.com>
    Signed-off-by: Stephen Hemminger <stephen@networkplumber.org>
---
 man/man8/tc-flower.8 | 23 +++++++++++------------
 1 file changed, 11 insertions(+), 12 deletions(-)

diff --git a/man/man8/tc-flower.8 b/man/man8/tc-flower.8
index dd35649..11cf774 100644
--- a/man/man8/tc-flower.8
+++ b/man/man8/tc-flower.8
@@ -107,8 +107,8 @@ or an unsigned 8bit value in hexadecimal format.
 Match on source or destination IP address.
 .I ADDRESS
 must be a valid IPv4 or IPv6 address, depending on
-.BR ether_type ,
-which has to be specified in beforehand.
+.BR protocol
+option of tc filter.
 .TP
 .BI dst_port " NUMBER"
 .TQ
@@ -129,16 +129,15 @@ is a 32 bit tunnel key id (e.g. VNI for VXLAN tunnel).
 must be a valid IPv4 or IPv6 address.
 .SH NOTES
 As stated above where applicable, matches of a certain layer implicitly depend
-on the matches of the next lower layer. Precisely, layer one and two matches (
-.BR indev , dst_mac , src_mac " and " eth_type )
-have no dependency, layer three matches (
-.BR ip_proto , dst_ip " and " src_ip )
-require
-.B eth_type
-being set to either
-.BR ipv4 " or " ipv6 ,
-and finally layer four matches (
-.BR dst_port " and " src_port )
+on the matches of the next lower layer. Precisely, layer one and two matches
+(\fBindev\fR,  \fBdst_mac\fR and \fBsrc_mac\fR)
+have no dependency, layer three matches
+(\fBip_proto\fR, \fBdst_ip\fR and \fBsrc_ip\fR)
+depend on the
+.B protocol
+option of tc filter
+and finally layer four matches
+(\fBdst_port\fR and \fBsrc_port\fR)
 depend on
 .B ip_proto
 being set to either
-- 
1.8.3.1

